lib/coderay/encoders/latex_encoder.rb

```diff
@@ -0,0 +1,521 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Emacs adds this line automatically when it discovers UTF-8 characters in the source file (even if they’re in the comments). The line is not “needed” in that sense thus.

Vale,
Quintus